homeassistant/components/izone/.translations/ca.json

```diff
@@ -0,0 +1,15 @@
+{
+    "config": {
+        "abort": {
+            "no_devices_found": "No s'han trobat dispositius de iZone a la xarxa.",
+            "single_instance_allowed": "Nom\u00e9s cal una sola configuraci\u00f3 de iZone."
```

Comment:
I don't think we should upload the .translations files ourselves - the translation files are automatically uploaded and will be overwritten automatically.

Reply:
I was wondering about that. The ones that are present are the same as daikin, only with the word daikin replaced by iZone. I put them in as a separate check in so easy enough to drop if desired. Alternatively they could just stay as they are until replaced. I don't know if the system is available outside Australia anyhow.

Reply:
We should not include translations except the default English that is generated from strings.json when running the translation script.